dockers/nvidia/Dockerfile

```diff
@@ -38,6 +38,8 @@ RUN \
     fi && \
 # save the examples \
     ls -lh lightning/ && \
+    rm -rf lightning/.git && \
```

Comment:
do we need to touch nvidia files?

Reply:
these are upstream changes, as of now do not merge this one.